en/docs/guides/identity-verification/idvp-ui.md

```diff
@@ -0,0 +1,307 @@
+| default          | Renders a text input field.                                                          | Console      |
+
+Refer [validation.ts](https://github.com/wso2/identity-apps/blob/master/modules/validation/src/validation.ts) and 
+[dynamic-ui-helper.tsx](https://github.com/wso2/identity-apps/blob/master/apps/console/src/features/identity-verification-providers/components/forms/helpers/dynamic-ui-helper.tsx#L269) for more details on how the validations are performed.
+
+**Supported Attributes**
+
+Common Attributes - All Input Fields
+The following attributes are supported by every input field type.
+
+| Attribute       | Description                                                                                                                                                                 | Example                          |
+|-----------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------|
+| name            | Name of the input field. The name of the input field is used as the key of the configuration property.                                                                      | “token”                          |
+| type            | Input field type out of one of the supported input field types specified above. The “default” input type is used if the type is not specified.                              | “password”                       | 
+| label           | The label that should be displayed with the input field.                                                                                                                    | “Token”                          |
+| hint            | The hint associated with the input field.                                                                                                                                   | “The token obtained from Onfido” |
+| required        | Whether the input field is required or not.                                                                                                                                 | true                             |
+| defaultValue    | Default value for the input field.                                                                                                                                          | “default value”                  |
+| displayOrder    | Defines the display order the configuration properties. If the attribute is not defined, elements are ordered according to the order they are defined in the metadata.json. | 1                                |
+| dataComponentId | The data component id of the input field which can be used in test automation.                                                                                              | “idvp-config-settings-token”     |
+
+
+Common Attributes - Text and Numeric Input Field Types
```

Comment:
`Common Attributes - Text and Numeric Input Field Types` should be a heading.

Reply:
Resolved